src/web/server/locales/en/common.json

```diff
@@ -116,7 +116,7 @@
   "confirm": {
     "newClaimTitle": "Application complete",
     "updatedClaimTitle": "Application updated",
-    "subTitle": "You are entitled to\n£{{totalVoucherValue}} per week"
+    "subTitle": "You are entitled to\n£{{totalVoucherValue}} a week. Your first payment\nwill be £{{totalVoucherValueForFourWeeks}}."
```

Comment:
The prototype is using 'You're' instead of 'You are'

Reply:
The monetary values need to be bold.

Reply:
Fixed.